src/components/PasswordInputComponent.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <label v-if="!props.repeat" class="title">Пароль</label>
+    <label v-else class="title">Повторите пароль</label>
+    <input v-model="_password" type="password" @input="isInteracted = true" required />
+    <span
+      v-if="!props.repeat"
+      v-show="!passwordIsGiven && isInteracted"
+      class="invalidDataError"
+    >Введите пароль</span>
+    <span
+      v-if="props.repeat"
+      v-show="!passwordIsRepeated && isInteracted"
+      class="invalidDataError"
+    >Введите пароль еще раз</span>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { defineProps, ref, watch } from 'vue';
+import type { Ref } from 'vue';
+
+const props = defineProps({
+  repeat: Boolean,
+})
+
+import { ValueError } from '@/errors';
+import Password from '@/utils/password';
+
+import { storeToRefs } from 'pinia';
+
+import { useFormStateStore, useAuthorizationStore } from '@/store';
+const formStateStore = useFormStateStore();
+const authorizationStore = useAuthorizationStore();
+const {
+  isInteracted,
+  passwordIsGiven,
+  passwordIsRepeated,
+} = storeToRefs(formStateStore);
+const { password, passwordRepeat } = storeToRefs(authorizationStore)
```

Comment:
It should be in the another copmponent's logic

Reply:
Done another possible way